src/pyfreeleh/providers/google/sheet/wrapper.py

```diff
@@ -169,14 +171,50 @@ def _parse_cell(self, cell: Dict[str, str], col: Dict[str, str]) -> Any:
         if typ == "boolean":
             return cell["v"]
         elif typ == "number":
-            if "." in cell["f"]:
-                return float(cell["f"])
+            if "f" in cell:
+                if "." in cell["f"]:
+                    return float(cell["f"])
 
-            return int(cell["f"])
+                return int(cell["f"])
+
+            # Computed data doesn't have raw value, number returned from aggregation will doesn't have `f`.
+            return int(cell["v"])
         elif typ == "string":
             return cell["v"]
         elif typ in ["date", "datetime", "timeofday"]:
-            # return datetime.strptime(cell["f"], col["pattern"])
             return cell["f"]
 
         raise ValueError("cell type {} is not supported".format(typ))
+
+
+class GoogleSheetSession:
```

Comment:
I find this class a bit useless :thinking: 

It seems like a class that has no specific reason for its existence other than grouping a few data together for easier passing over to other classes?

Reply:
Yup, it's just a convenience wrapper.

Reply:
I think we don't need this session concept since it doesn't have any specific reason for its existence yet (at least as of now).

Reply:
Done.